api/app/services/ingestor/strategy/google_doc/strategy.rb

```diff
@@ -0,0 +1,53 @@
+require "filesize"
+
+module Ingestor
+  module Strategy
+    module GoogleDoc
+      # The <tt>Ingestor::Strategy::GoogleDoc</tt> class provides a strategy for ingesting
+      # Google Doc source documents into Manifold.
+      #
+      # @author Max Ono
+      class Strategy < Ingestor::Strategy::Base
+        include Ingestor::Loggable
+
+        def initialize(ingestion)
+          @ingestion = ingestion
+          @inspector = ::Ingestor::Strategy::GoogleDoc::Inspector::GoogleDoc.new(
+            @ingestion.source_path,
+            @ingestion.logger
+          )
+          @logger = @ingestion.logger ||
+                    Naught.build { |config| config.mimic Logger }
+        end
+
+        # Return true if the url is valid
+        def self.can_ingest?(ingestion)
+          inspector = ::Ingestor::Strategy::GoogleDoc::Inspector::GoogleDoc.new(
```

Comment:
DRY up your code into a private method. The 4 lines that return the inspector are repeated here and could be consolidated.